client/ayon_sitesync/providers/abstract_provider.py

```diff
@@ -76,7 +92,7 @@ def download_file(self, source_path, local_path,
             addon (SiteSyncAddon): addon instance to call update_db on
             project_name (str):
             file (dict): info about uploaded file (matches structure from db)
-            representation (dict): complete repre containing 'file'
+            repre_status (dict): complete repre containing 'file'
```

Comment:
Either the new argument name is odd or the description odd.

Also, I'm confused as to why the arguments in this docstring are not sorted in the order of the actual argument. It's to explain that the bottom ones are for saving progress - but do IDEs not get confused?

Reply:
Updated.

Reply:
BTW the same text at multiple places.

Reply:
Updated missed.